src/main/cpp/include/api/genomicsdb.h

```diff
@@ -508,5 +508,7 @@ GENOMICSDB_EXPORT const genomicsdb_variant_call_t* GenomicsDBResults<genomicsdb_
 template<>
 GENOMICSDB_EXPORT void GenomicsDBResults<genomicsdb_variant_call_t>::free();
 
+template<class VariantOrVariantCall>
```

Comment:
Does this work without the `GENOMICSDB_EXPORT` macro?

Reply:
I think so, because I don't need this to be visible for the shared library, just to link between the object files. If you prefer I could try moving these declarations back into the .cpp files...I think the crux of the fix is those specialized template declarations.